packages/m2m-gateway/src/services/eserviceService.ts

```diff
@@ -242,6 +242,34 @@ export function eserviceServiceBuilder(
         throw eserviceDescriptorNotFound(eServiceId, descriptorId);
       }
 
+      return toM2MGatewayApiEServiceDescriptor(descriptor);
+    },
+    async publishDescriptor(
+      eServiceId: EServiceId,
```

Comment:
Consistency in this file
```suggestion
      eserviceId: EServiceId,
```